tests/test_pages/test_code.py

```diff
@@ -860,8 +862,8 @@ def test_html_img(self):
         '<a src="data:text/html;base64,%s"</a>' % evil_b64_data,
         '<img src="https://Evil.com">',
```

Comment:
Sorry, this is a debug leftover in a way, or at least it's a non-`data` URL in a list of data URLs, which is weird.

Reply:
nvm.